BankSDK/GiniBankSDK/Sources/GiniBankSDK/Extensions/Foundation/NumberFormatter+Utils.swift

```diff
@@ -0,0 +1,34 @@
+//
+//  NumberFormatter+Utils.swift
+//
+//  Copyright © 2024 Gini GmbH. All rights reserved.
+//
+
+import Foundation
+
+extension NumberFormatter {
+    private static func makeLocalizedDecimalFormatter(locale: Locale = .current,
+                                                      numberStyle: NumberFormatter.Style = .decimal,
+                                                      usesGroupingSeparator: Bool = true) -> NumberFormatter {
+        let formatter = NumberFormatter()
+        formatter.locale = locale
+        formatter.numberStyle = numberStyle
+        formatter.usesGroupingSeparator = usesGroupingSeparator
+        return formatter
+    }
+
+    static var skontoPriceFormatter: NumberFormatter {
```

Comment:
[suggestion]: I propose to rename this into something more generic like: `twoDecimalPriceFormatter` since it is part of an extension should not have a very specific name based on a feature.